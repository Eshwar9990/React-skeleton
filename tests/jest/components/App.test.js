import React from 'react';
import App from '../../../src/components/App';

describe('Positive Workflow', () => {
  it('should render App', () => {
    const mockFunc = jest.fn();
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
