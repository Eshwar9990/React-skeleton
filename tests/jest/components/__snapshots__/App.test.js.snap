// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Positive Workflow should render App 1`] = `
<div>
  Hello World
</div>
`;
